drivers/hid/ithc/ithc-api.h

```diff
@@ -0,0 +1,16 @@
+struct ithc_api {
+	struct ithc *ithc;
+	struct ithc_dma_rx *rx;
+	struct miscdevice m;
+	atomic_t open_count;
+};
+
+struct ithc_api_header {
+	u8 hdr_size;
+	u8 reserved[3];
+	u32 msg_num;
+	u32 size;
+};
```

Comment:
Should this also be `__packed`?

Reply:
This is the old non-HID API. I've kept it around for debugging, but I'll remove all of it in the near future.